packages/dbml-core/src/parse/ANTLR/ASTGeneration/postgres/PostgresASTGen.js

```diff
@@ -804,6 +804,29 @@ export default class PostgresASTGen extends PostgreSQLParserVisitor {
             cmd.value.endpoints[0].schemaName = schemaName;
             this.data.refs.push(cmd.value);
             break;
+          case TABLE_CONSTRAINT_KIND.UNIQUE:
+          case TABLE_CONSTRAINT_KIND.INDEX:
+            if (cmd.value.columns == 0) break;
```

Comment:
I think you mean columns.length? 🤔 
```suggestion
            if (cmd.value.columns.length === 0) break;
```

Reply:
fixed